107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -129,7 +129,7 @@ public void jsr107LoaderGetsRegistered() {
     merger.mergeConfigurations("cache", configuration);
 
     assertThat(factory.called, is(true));
-    verify(cacheLoaderWriterFactory).registerJsr107Loader(eq("cache"), Matchers.<CacheLoaderWriter<Object, Object>>anyObject());
+    verify(cacheLoaderWriterFactory).registerJsr107Loader(eq("cache"), ArgumentMatchers.<CacheLoaderWriter<Object, Object>>any());
```

Comment:
Right. Since the parameter type is already expecting this type, I will use `isNotNull` which I find more explicit.